lib/win32/certstore/mixin/helpers.rb

```diff
@@ -0,0 +1,85 @@
+#
+# Author:: Vasundhara Jagdale (<vasundhara.jagdale@msystechnologies.com>)
```

Comment:
Could you also include the Author/s from `Chef::Mixin::WideString`

Reply:
yes probably I should add those name my bad.